### PR TITLE
Add Webpack to front #21 #8

### PR DESCRIPTION
**Pull request by Hairash**
* Tests and project work correct
* Add local webserver
* Put files in folders
* Add PixiJS dependency

Problems:
* Code in browser is unreadable

### CODE REVIEW

File: frontend/src/grid.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+import * as PIXI from 'pixi.js';
+
 class Hex extends PIXI.Graphics {
-    constructor(side, border_width, border_color) {
+    constructor(side,
+        border_width,    border_color) {
```
Comment (alex-eg) [inline frontend/src/grid.js:5]: 🤔

File: frontend/src/grid.js [hunk 2]
```diff
@@ -111,4 +114,4 @@ class Hex_grid {
     }
 }
 
-
+// module.exports = { Hex, Hex_grid };
```
Comment (alex-eg) [inline frontend/src/grid.js:117]: Commented code.

File: frontend/src/grid.js (hunk 1 above)
Comment (Hairash) [reply frontend/src/grid.js:5]: hah, forgot to change it back, thanks

File: frontend/src/grid.js (hunk 2 above)
Comment (Hairash) [reply frontend/src/grid.js:117]: ok

**Pull request merged**